orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/StageBuilder.groovy

```diff
@@ -126,6 +127,28 @@ abstract class StageBuilder implements ApplicationContextAware {
                task.status = ExecutionStatus.NOT_STARTED
              }
          }
+
+    def childStages = stage.execution.stages.findAll {
+      def requisiteStageRefIds = it.requisiteStageRefIds ?: []
+      if (it.context.requisiteIds) {
```

Comment:
`it.context.requisiteIds` is what `WaitForRequisiteCompletionStage` does (it doesn't use `it.requsiteStageRefIds` for obvious reasons).